dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsolute.java

```diff
@@ -1,19 +1,28 @@
 package com.dotmarketing.startup.runonce;
 
+import com.dotcms.contenttype.transform.JsonTransformer;
 import com.dotcms.rendering.velocity.viewtools.DotTemplateTool;
 import com.dotmarketing.common.db.DotConnect;
 import com.dotmarketing.common.db.Params;
 import com.dotmarketing.db.DbConnectionFactory;
 import com.dotmarketing.exception.DotDataException;
 import com.dotmarketing.exception.DotRuntimeException;
 import com.dotmarketing.portlets.containers.business.FileAssetContainerUtil;
-import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
+import com.dotmarketing.portlets.templates.design.bean.ContainerUUID;
+import com.dotmarketing.portlets.templates.design.util.DesignTemplateUtil;
```

Comment:
We still need to get rid of these two dependencies. Let's either copy what's needed and make it internal/private to the UT

Reply:
done 
https://github.com/dotCMS/core/pull/22604/commits/43ccd6f6d034c348870f47b42896b5e2e68dce1b https://github.com/dotCMS/core/pull/22604/commits/36739572f0e4510c216908e3b820e1e095ca9efa